recruitment/models.py

```diff
@@ -101,10 +101,8 @@ def handle_answers_from_request(self, request, user):
                         user=user
                     )
                     answer_string = request.POST[key]
-
-                    if answer_string:
-                        answer.answer = answer_string
-                        answer.save()
+                    answer.answer = answer_string
+                    answer.save()
```

Comment:
The answer_string is a relic from the era of the buggy if-statement. I agree it should be removed.

The custom field answers are always shown in the UI using custom_fields.html with the answers from questions_with_answers_for_user and stored to the database using handle_answers_from_request in the ExtraField class. These functions are heavily coupled since they need to use the exact same post keys. They will always generate form elements for all answers so there won't be partial submission. 

If one decides to use the CustomFieldAnswer class in any other way, for instance creating forms with just a subset of the CustomFields, they should write both the view and presentational logic as they like and not the above functions.